### ISSUE TITLE
Validation on ngWig not working

### ISSUE THREAD

**Issue by LoganDupont** [stevermeister/ngWig#158]
The validation does not work because a <p> element is inserted by default if there is no value of the model. So we have problems for the required & minLength/maxLength due the paragraph tags. We had an work around but because it's an rich text editor we cannot filter on html-tags. So is it possible to find a solution for this problem.

Also if we want to save the model in the db the paragraph tags are added at the end and we can't also filter on html-tags for this one otherwise we lose information that the user has entered inside the ng-wig.

Greets,
Logan

**Comment by bampakoa** [comment]
Dupe of #149

**Comment by LoganDupont** [comment]
Thats true, but that issue was only for the required validation that's why i created a more general one. Because the <p>-tags gives you also problems for other validations and your model.  So i think it should be the other way around.
